Gemfile

```
@@ -21,17 +21,17 @@ gem "sinatra-contrib", path: "sinatra-contrib"
 gem "twitter-text", "1.14.7"
 
 
-gem "activesupport", "~> 5.1.6", platforms: [ :jruby, :mri ]
+gem "activesupport", "~> 5.1.6"
 
-gem 'redcarpet', platforms: [ :mri ]
+gem 'redcarpet', platforms: [ :ruby ]
 gem 'wlang', '>= 3.0.1'
-gem 'bluecloth', platforms: [ :mri ]
-gem 'rdiscount', platforms: [ :mri ]
-gem 'RedCloth', platforms: [ :mri ]
-gem 'puma', platforms: [ :jruby, :mri ]
-gem 'yajl-ruby', platforms: [ :mri ]
-gem 'nokogiri', '> 1.5.0', platforms: [ :jruby, :mri ]
-gem 'rainbows', platforms: [ :mri ]
+gem 'bluecloth', platforms: [ :ruby ]
+gem 'rdiscount', platforms: [ :ruby ]
+gem 'RedCloth', platforms: [ :ruby ]
+gem 'puma', platforms: [ :jruby, :ruby ]
+gem 'yajl-ruby', platforms: [ :ruby ]
+gem 'nokogiri', '> 1.5.0', platforms: [ :jruby, :ruby ]
```

```suggestion
gem 'nokogiri', '> 1.5.0'
```
since it's supported for all 3 Rubies

---

Perfect, thanks!